gms/gossiper.cc

```diff
@@ -1616,6 +1617,24 @@ void gossiper::mark_alive(inet_address addr) {
 }
 
 future<> gossiper::real_mark_alive(inet_address addr) {
+    co_await utils::get_local_injector().inject_with_handler("raft_group_registry::on_alive", [this, endpoint = addr] (auto& handler) -> future<> {
```

Comment:
it's better to rename the injection name, e.g. `gossiper::real_mark_alive`

Reply:
no problem

Reply:
done in v5